hw/top_earlgrey/ip/ast/ast.core

```diff
@@ -114,4 +114,3 @@ targets:
       vcs:
         vcs_options: [-timescale=1ns/1ps -l vcs.log]
     toplevel: ast
-
```

Comment:
is this a rebase artifact? this commit appears to be empty otherwise...

Reply:
damn it. it is. let me fix that.